docs/pages/database-access/reference/configuration.mdx

```diff
@@ -111,6 +111,12 @@ spec:
     ...
     -----END CERTIFICATE-----
 
+  # MySQL only options.
+  mysql:
+    # The default MySQL server version reported by Teleport Proxy.
+    # When this option is set the Database Agent doesn't try to check the MySQL server version.
```

Comment:
```suggestion
    # When this option is set, the Database Service does not try to check the MySQL server version.
```

Reply:
And the Database Service assumes that the server version is the one provided?

Reply:
Yes, this is a fallback version that will be used until we're able to get the version from the database. In most cases this value will never be used. If database is down during during Teleport startup then we will use this version for very first connection, as we were not able to fetch the version earlier.